### PR TITLE
Feature/introduction

### LINKED ISSUE #8: Introduction - molecule
[InVision](https://invis.io/N4FDSKWX8ZB#/279353742_Introduction)

### PR DESCRIPTION
**Pull request by knuch**
fix #8

only question left: Is the text width smaller then the global width ? If yes, then we need to adapt the scss around `@extend .container;`

### CODE REVIEW

File: assets/components/molecules/introduction/introduction.scss [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    .col-md-12 {
+      height: 100%;
+    }
+  }
+
+  .introduction {
+    @extend .container;
+
+    @include media-breakpoint-down(sm) { 
+      background: red;
```
Comment (Yago) [inline assets/components/molecules/introduction/introduction.scss]: giphy

File: assets/components/molecules/introduction/introduction.scss [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  flex-flow: column nowrap;
+  align-items: center;
+  justify-content: center;
+  padding: 3rem 0 2.5rem;
+  background: $gray-200;
+
+  @include media-breakpoint-down(sm) {
+    min-height: 100vh;
+
+    .col-md-12 {
```
Comment (Yago) [inline assets/components/molecules/introduction/introduction.scss]: I don't see any `.col-md-12` in your component, but you should definitely use a `.container`

File: assets/config/bootstrap-variables.scss [hunk 3]
```diff
@@ -11,7 +11,7 @@
 // stylelint-disable
 $white:    #fff !default;
 $gray-100: #f8f9fa !default;
-$gray-200: #e9ecef !default;
+$gray-200: #EEEEEE !default;
```
Comment (Yago) [inline assets/config/bootstrap-variables.scss]: Is that a validated color ?

File: assets/config/bootstrap-variables.scss [hunk 4]
```diff
@@ -244,7 +244,7 @@ $font-weight-base:            $font-weight-normal !default;
 $line-height-base:            1.47 !default;
 
 $h1-font-size:                $font-size-base * 3.17 !default;
-$h2-font-size:                $font-size-base * 2.47 !default;
+$h2-font-size:                $font-size-base * 2.33 !default;
```
Comment (Yago) [inline assets/config/bootstrap-variables.scss]: It's not what is writen in the `docs/typography.html`and I just saw that Guillaume update everything on [InVision](https://projects.invisionapp.com/share/N4FDSKWX8ZB#/screens/274002805) 😭 

I will upload those sizes outside this pull-request 😉

Comment (Yago) [reply assets/config/bootstrap-variables.scss]: *update : I juste update all the font-sizes in `dev` (9303402a44f2bf37dfd00a0c385756099a2e9940)

File: assets/config/bootstrap-variables.scss (hunk 3 above)
Comment (Yago) [reply assets/config/bootstrap-variables.scss]: *update : I juste update and document the “validated” colors in `dev` (0b4f22cf136029f2c36bfff61aa331b9267fa703)

**Comment by knuch** [comment]
@Yago Changes done :)

**Comment by Yago** [comment]
I remove `.col-12` which are the default behaviour and it's all right !

**Comment by Yago** [comment]
gif

**Pull request merged**